src/pages/Aboutpage.js

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect } from 'react';
+import AOS from 'aos';
+import 'aos/dist/aos.css';
+import TeamMember from '../components/TeamMember';
+const Aboutpage = () => {
+  useEffect(() => {
+    AOS.init({
+      duration: 2000,
+    });
+  });
+  return (
+    <>
+      {' '}
+      <div
+        style={{
```

Comment:
lets not use inline styles and instead put this line in a separate css file in the same folder